wpilibj/src/main/java/edu/wpi/first/wpilibj/motorcontrol/MotorController.java

```diff
@@ -37,6 +37,13 @@ default void setVoltage(double outputVolts) {
    */
   double get();
 
+  /**
+   * Common interface for getting the current voltage of the motor controller.
+   *
+   * @return The current controller voltage output. Value is between 0V and 13V.
```

Comment:
```suggestion
   * @return The current controller voltage output. Value is between 0V and 12V.
```

Reply:
It can exceed 12v if battery voltage is above 12v, right?

Reply:
Yes, but 13 V is unusual. Perhaps "0V and battery voltage (nominally 12 V)" is better.